app/graphql/schema.graphql

```diff
@@ -10,6 +10,7 @@ type Benchmark implements Node {
   profiles: [Profile!]
   refId: String!
   rules: [Rule!]
+  supportedProfiles: [Profile!]
```

Comment:
Should this be an instance/object property? Shouldn't this be global (i.e. a separate query)?

Reply:
Second issue: it returns canonical profiles only. It might be named accurately, e.g. `supportedCanonicalProfiles`.

Reply:
Your proposal makes sense, I just wanted to have it simpler here, but I can change it.

Well, `profiles` also returns just canonical profiles.

Reply:
Can it be moved to a standalone query, like the `latest_benchmarks` are?
https://github.com/RedHatInsights/compliance-backend/blob/f9fadd1faf51425156d693a16c358c24b825095c/app/graphql/types/query.rb#L63